tests/driver_isl29125/main.c

```diff
@@ -76,7 +76,7 @@ int main(void)
         "ISL29125_MODE_R", "ISL29125_MODE_G", "ISL29125_MODE_B",
         "ISL29125_MODE_RG", "ISL29125_MODE_GB"};
 
-    for (size_t i = 0; i < sizeof(modes); i++) {
+    for (size_t i = 0; i < sizeof(modes)/sizeof(modes[0]); i++) {
```

Comment:
Please add spaces before and after operators:

```C
sizeof(modes) / sizeof(modes[0])
```

Reply:
Done